### ISSUE TITLE
Too early stop

### ISSUE THREAD

**Issue by mxzhao** [robot0321/DepthRegularizedGS#15]
I tried the preprocessed dataset and used the method 2 to train fern and got too early stop at iteration 1800:
command:
python train.py -s E:\PublicData\3D\nerf\nerf_llff_fewshot_resize\fern --eval --port 6312 --model_path .\output\method2_fern --resolution 1 --kshot 5 --seed 3 --depth --usedepthReg

Optimizing .\output\method2_fern
Output folder: .\output\method2_fern
Reading camera 2/20img_size [384, 512]
Using cache found in C:\Users\zmx/.cache\torch\hub\intel-isl_MiDaS_master
C:\Users\zmx\AppData\Local\anaconda3\envs\gs-avatar\lib\site-packages\torch\functional.py:504: UserWarning: torch.meshgrid: in an upcoming release, it will be required to pass the indexing argument. (Triggered internally at C:\cb\pytorch_1000000000000\work\aten\src\ATen\native\TensorShape.cpp:3484.)
  return _VF.meshgrid(tensors, **kwargs)  # type: ignore[attr-defined]
Params passed to Resize transform:
        width:  512
        height:  384
        resize_target:  True
        keep_aspect_ratio:  True
        ensure_multiple_of:  32
        resize_method:  minimal
Using pretrained resource url::https://github.com/isl-org/ZoeDepth/releases/download/v1.0/ZoeD_M12_NK.pt
Loaded successfully
ITER=  1000 loss=  0.0007, params=[8.7718,4.9724], lr=  0.1076
loss =   0.00073
Reading camera 3/20ITER=  1000 loss=  0.0007, params=[9.6640,4.2370], lr=  0.1076
loss =   0.00069
Reading camera 5/20ITER=  1000 loss=  0.0005, params=[9.3534,4.4116], lr=  0.1076
loss =   0.00050
Reading camera 11/20ITER=  1000 loss=  0.0005, params=[9.4468,1.7163], lr=  0.1076
loss =   0.00049
Reading camera 20/20ITER=  1000 loss=  0.0005, params=[9.8946,1.3672], lr=  0.1076
loss =   0.00048

Loading Training Cameras
Loading Test Cameras
Number of points at initialisation :  5325
Training progress:   6%|▋           | 1800/30000 [00:47<22:44, 20.67it/s, Loss=0.2580238, Deploss=0.1356, #pts=1424853]C:\Users\zmx\AppData\Local\anaconda3\envs\gs-avatar\lib\site-packages\torchvision\models\_utils.py:135: UserWarning: Using 'weights' as positional parameter(s) is deprecated since 0.13 and may be removed in the future. Please use keyword parameter(s) instead.
  warnings.warn(
C:\Users\zmx\AppData\Local\anaconda3\envs\gs-avatar\lib\site-packages\torchvision\models\_utils.py:223: UserWarning: Arguments other than a weight enum or `None` for 'weights' are deprecated since 0.13 and may be removed in the future. The current behavior is equivalent to passing `weights=AlexNet_Weights.IMAGENET1K_V1`. You can also use `weights=AlexNet_Weights.DEFAULT` to get the most up-to-date weights.
  warnings.warn(msg)
Downloading: "https://raw.githubusercontent.com/richzhang/PerceptualSimilarity/master/lpips/weights/v0.1/alex.pth" to C:\Users\zmx/.cache\torch\hub\checkpoints\alex.pth
100%|█████████████████████████████████████████████████████████████████████████████| 5.87k/5.87k [00:00<00:00, 1.81MB/s]
[ITER 1800] Evaluating test: L1 0.067284 PSNR 19.49                                        | 0.00/5.87k [00:00<?, ?B/s]
Training progress:   6%|▋           | 1800/30000 [01:00<22:44, 20.67it/s, Loss=0.2580238, Deploss=0.1356, #pts=1424853][ITER 1800] Evaluating train: L1 0.030368 PSNR 25.19
!!! Stop Point: 1800 !!!
Training progress:   6%|▋           | 1800/30000 [01:05<17:08, 27.41it/s, Loss=0.2580238, Deploss=0.1356, #pts=1424853]

Training complete.

I also tried flower, it stopped at 1900.

I think there should be some issues, why? I didn't make any code change.

thanks
BR
Mingxi

**Comment by duyanxi** [comment]
have you solved this problem yet?

**Comment by JX-Github-web** [comment]
have you solved this problem yet?
![image](https://github.com/user-attachments/assets/322795c1-e5a6-4e59-9f5a-c856be5c7740)
